torch/distributions/binomial.py

```diff
@@ -0,0 +1,88 @@
+from numbers import Number
+import torch
+import math
+from torch.distributions import constraints
+from torch.distributions.distribution import Distribution
+from torch.distributions.utils import broadcast_all, probs_to_logits, lazy_property
+from torch.distributions.utils import clamp_probs
+from torch.autograd import Variable
+
+
+class Binomial(Distribution):
+    r"""
+    Creates a Binomial distribution parameterized by `total_count` and
+    either `probs` or `logits` (but not both).
+
+    -   :meth:`sample` requires a single shared `total_count` for all
+        parameters and samples.
+
+    Example::
+
+        >>> m = Binomial(100, torch.Tensor([0 , .2, .8, 1]))
+        >>> x = m.sample()
+         0
+         22
+         71
+         100
+        [torch.FloatTensor of size 4]]
+
+    Args:
+        total_count (int): number of trials
+        probs (Tensor or Variable): event probabilities
+        logits (Tensor or Variable): event log probabilities
+    """
+    params = {'probs': constraints.unit_interval}
+    has_enumerate_support = True
+
+    def __init__(self, total_count=1, probs=None, logits=None):

```

Comment:
Weren't we using log-odds for `logits` of `Bernoulli`? Is this consistent with TensorFlow?

Reply:
Yep, my bad, just fixed it.

Reply:
@fritzo this is exactly why I wanted to use a different name. Calling two semantically different things the same is going to end up in an error sooner or later without a type system

Reply:
Fair enough. Should we switch to `log_probs` and `log_odds` then?